packages/sysdig/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,289 @@
+---
+description: Pipeline for mapping Sysdig fields to ECS / sysdig-namespaced fields
+processors:
+- set:
+    field: event.kind
+    value: 'alert'
+- json:
+    field: message
+    target_field: json
+    if: ctx?.message != null
```

```suggestion
    if: ctx.message != null
```

`ctx` is assumed that always exist, so can you replace every `ctx?` by just `ctx`?

---

Makes sense to me - updated